contracts/swaps/connectors/SwapsImplSovrynSwap.sol

```diff
@@ -54,7 +54,6 @@ contract SwapsImplSovrynSwap is State, ISwapsImpl {
 	 *
 	 * @param sourceTokenAddress The address of the source tokens.
 	 * @param destTokenAddress The address of the destination tokens.
-	 * @param receiverAddress The address to receive the swapped tokens.
```

Comment:
why is this param removed from comments?

Reply:
It was removed by @smbsp in previous commit, forgot to added it back as in we added that params back again